molior-project-manager

```diff
@@ -268,7 +268,7 @@ COOKIE_FILE="$(mktemp)"
 molior_session_call()
 {
   URL_PATH="$1" && shift
-  curl $MOLIOR_URL$URL_PATH -s --retry 10 \
+  curl $MOLIOR_URL$URL_PATH -n -s --retry 10 \
```

Comment:
I'm not sure if we should add the `-n` here, because a few lines further down we explicitly pass the `$MOLIOR_USERNAME` and `$MOLIOR_PASSWORD` to curl (matching .netrc credentials will likely override these)